composer.json

```diff
@@ -71,7 +73,7 @@
   "extra": {
     "branch-alias": {
       "dev-master": "1.0-dev",
-      "dev-develop": "1.1-dev"
+      "dev-prooph_v7": "0.3-dev"
```

Comment:
I was a bit in struggle with these dependecies setup. Maybe the alias should be removed, true.